src/reporters/base-reporter.js

```diff
@@ -37,7 +37,8 @@ export function stringifyLangArgs(args: Array<any>): Array<string> {
       return val.inspect();
     } else {
       try {
-        return JSON.stringify(val) || val + '';
+        let str = JSON.stringify(val) || val + '';
+        return str.replace(/((?:^|[^\\])(?:\\{2})*)\\u001[bB]/g, '$1\u001b');
```

Comment:
simpler regex or an instruction how to read it would definitely be appreciated :)
Also a link to the issue would be great for people to have context why the change was made.